csrankings-y.csv

```diff
@@ -865,6 +867,7 @@ Yuval Emek,Technion,https://yemek.net.technion.ac.il,q2WvHtkAAAAJ
 Yuval Filmus,Technion,http://www.cs.technion.ac.il/people/yuvalfi,TFvs8NgAAAAJ
 Yuval Ishai,Technion,http://www.cs.technion.ac.il/~yuvali,zc920lAAAAAJ
 Yuval Kochman,Hebrew University of Jerusalem,http://www.cs.huji.ac.il/~yuvalko,gNoj8SkAAAAJ
+Yuval Peled,Hebrew University of Jerusalem,https://www.cs.huji.ac.il/~yuvalp02/,l5OiPlcAAAAJ

```

Comment:
Says moved to NYU (https://www.cs.huji.ac.il/w~yuvalp02/); please update.

Reply:
Bump @ShaiYer

Reply:
Sorry, didn't realized that you are waiting for my response.
We fixed the website of Yuval Peled - He is back from NYU to the Hebrew University.

Should I create a new pull request?